tests/schema/__app/generated/schema.graphql

```diff
@@ -33,8 +33,8 @@ input BubbleMembersWhereInput {
 input BubbleWhereInput {
   AND: [BubbleWhereInput!]
   createdAt: DateTimeFilter
-  id: UUIDFilter
-  members: UserFilter
+  id: StringFilter
+  members: UserListRelationFilter
```

Comment:
BREAKING CHANGE to document

The naming pattern here has changed. On the surface, here, it appears to go from `<ModelName>Filter` to `<ModelName>ListRelationFilter`. But maybe the `List` and/or `Relation` parts are variable too.

Reply:
I added some new models and patterns to the integration test to verify things. It seems the change is as simple as RM_Filter to RM_ListRelationFilter.